tabular/src/autogluon/tabular/models/rf/rf_quantile.py

```diff
@@ -334,15 +334,15 @@ def __init__(
         )
 
 
-class ExtraTreeQuantileRegressor(ExtraTreeRegressor, BaseTreeQuantileRegressor):
+class ExtraTreeQuantileRegressor(BaseTreeQuantileRegressor, ExtraTreeRegressor):
     def __init__(
         self,
         criterion="squared_error",
         splitter="random",
         max_depth=None,
         min_samples_split=2,
         min_samples_leaf=1,
-        max_features="auto",
+        max_features=1.0,
```

Comment:
What was the future warning? Isn't this a change in the model that would change results?

Reply:
The "auto" value has been deprecated and 1.0 is the new default, here is the relevant announcement
https://github.com/scikit-learn/scikit-learn/blob/f44291b775cb67998c0965a72a8eeebc0beed37b/doc/whats_new/v1.1.rst?plain=1#L1274

Reply:
Gotcha, makes sense!